include/nuttx/wireless/bluetooth/bt_ioctl.h

```diff
@@ -61,9 +61,13 @@
 #define HCI_GATTRD_DATA     32   /* Max number of bytes in GATT read data */
 #define HCI_GATTWR_DATA     16   /* Max number of bytes in GATT write data */
 
+#ifndef CONFIG_BLUETOOTH_MAXSCANDATA
```

Comment:
Why add this?

Reply:
if user just do btsnoop open/close  ioctl without any wireless bluetooth host operation， which would BB  in  line:297. 

because  BLUETOOTH_MAXSCANDATA depends on WIRELESS_BLUETOOTH_HOST，which would expand many unused config.

struct bt_scanresponse_s
{
  bt_addr_le_t sr_addr;             /* Advertiser LE address and type */
  int8_t sr_rssi;                   /* Strength of advertiser signal */
  uint8_t sr_type;                  /* Type of advertising response */
  uint8_t sr_len;                   /* Length of advertiser data */
  uint8_t sr_data[CONFIG_BLUETOOTH_MAXSCANDATA];
};

Reply:
Ok